README.md

```diff
@@ -0,0 +1,63 @@
+## Cloudify ARIA Extensions
+
+This repository enables ARIA to utilize some of Cloudify's abilities, such as orchestrating using AWS and Openstack.
+
+### Installing
+
+1. clone the repository
+
+`git clone https://github.com/cloudify-cosmo/cloudify-aria-extensions.git`
+
+2. install the requirements:
+
+`pip install -r requirements.txt`
+
+3. install Cloudify ARIA extensions:
```

Comment:
Clarify this is for installing the adapter, and that this should be installed in the same environment ARIA is installed